transport/grpc/options.go

```diff
@@ -119,6 +119,15 @@ func ClientMaxSendMsgSize(clientMaxSendMsgSize int) TransportOption {
 	}
 }
 
+// ClientTLS says to use TLS with the system certificate pool on the client side.
+//
+// The default is to not use TLS.
+func ClientTLS() TransportOption {
```

Comment:
This can be adjusted to take a `tls.Config` via https://godoc.org/google.golang.org/grpc/credentials#NewTLS instead of via https://godoc.org/google.golang.org/grpc/credentials#NewClientTLSFromCert.

Reply:
If we do this, I'm not sure what we do in terms of the `transportSpec` in `transport/grpc/config.go`.